app/assets/javascripts/users.js

```diff
@@ -29,3 +29,31 @@ function get_select_values(select) {
 
   return result;
 }
+function test_mail(item, id){
+  $("#mail_message").remove();
+  $('#test_indicator').show();
+  $(item).addClass("disable-link");
+  var message = $("<p></p>");
+  message.attr("id","mail_message");
+  var data =$("form").data('id');
+  $.ajax({
+    url: foreman_url("users/" + id + "/test_mail"),
+    type: 'put',
+    data: data,
+    success: function(result, textstatus, xhr) {
+      message.text(result.message);
+      message.addClass("alert alert-success");
+    },
+    error: function (xhr) {
+      var response = xhr.responseText;
+      message.append("<b> Error:  </b>");
+      message.append(response);
+      message.addClass("alert alert-danger");
+    },
+    complete:function (result) {
+      $('#test_indicator').hide();
+      $(item).removeClass("disable-link");
+      $("#mail_preferences").prepend(message);
+    }
+  })
+}
```

Comment:
Hello, thanks for the contribution.

First, why don't we use our info-bubbles, we already have these in the app. Instead creating your own element, you could re-use existing code:

```js
notify("<p>test</p>", 'success');
notify("<p>test</p>", 'error');
```